docs/hubs/expense/api-documentation.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Expense APIs
+seo: Expense API Docs | Cloud Elements API Docs
+title: Platform Documentation
+description: Expense Hub API Documentation.
+layout: sidebarapidocs
+apis: API Docs
+platform: expense
+breadcrumbs: /docs/hubs/hub-docs.html
+parent: Back to Hub Docs
+order: 90
+---
+
+{% include swagger.html %}
+
+{% include padding-bottom.html %}
+{% include padding-bottom.html %}
```

Comment:
Good question. That was in whatever I copied and I didn't intentionally do it. How do we define space otherwise? I'd assume it's in the layout.